src/Time/Time.php

```diff
@@ -82,8 +139,10 @@ public static function timestampToGmt($time = null)
         $time = $time ?? time();
         $dateTime = new \DateTime();
         $dateTime->setTimestamp($time)->modify('+2 hours');
+        $format = isset($this->dateFormat) ? $format : 'd/m/Y H:i:s';
```

Comment:
We can consider letting the `$this->dateFormat` be the default date time format string.

Then this line should be unnecessary.

Reply:
So it mean it should has default value as well, or user should need to set it?

Reply:
That's right.